src/common/docker-compose/index.ts

```diff
@@ -71,6 +75,10 @@ export const generate = (dependency_manager: DependencyManager, build_prod = fal
           compose.services[node.normalized_ref].command = debug_options.command;
```

Comment:
Can you add entrypoint to the getDebugOptions. Would be nice to be able to override in the env.json.